dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/IEEE_PSS2A.mo

```diff
@@ -0,0 +1,128 @@
+within Dynawo.Electrical.Controls.Machines.PowerSystemStabilizers;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com) and UPC/Citcea (https://www.citcea.upc.edu/)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model IEEE_PSS2A "Power System Stabilizer"
```

Comment:
Can we include a bit more accurate comment on the model? I didn't find any official reference to refer to.